.eslintrc.json

```diff
@@ -23,6 +23,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".jsx", ".tsx"] }],
     "react/react-in-jsx-scope": "off",
     "react/no-unescaped-entities": 0,
+    "@react/no-array-index-key": "off",
```

Comment:
this rule should not be active, react doesn't work well with index as key.
https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys/56781201#56781201

instead, try using unique values from the object you're iterating as a key.

Reply:
just revert this. we will use uuid for keys to solve the issue.
We should create separate task for this

Reply:
I think it was okay to use index here, since we wouldn't mutate the array in any way. But safer and better with unique keys.